submissions/fatimaOlasunkanmiOjo/html-forms/contact.html

```diff
@@ -7,6 +7,7 @@
     <title>Contact Page</title>
 </head>
 <body>
+    <a href="index.html">Product</a>
```

Comment:
You should have two links here. The link to the contact page and the link to the product page.